ssg/build_remediations.py

```diff
@@ -204,6 +192,32 @@ def parse_from_file(file_path, env_yaml):
     return remediation(mod_file, config)
 
 
+def parse_from_file(file_path, env_yaml):
```

Comment:
Obviously, the naming suggestion apply here as well.

A Remediation class would solve the situation and avoid too long names, as it would read files either with or without jinja expansion, and it would have methods for metadata extraction.